### ISSUE TITLE
remove `imeta rmw` (and `addw`?) (and `lsw`?) (and `adda`?)

### ISSUE THREAD

**Issue by mstfdkmn** [irods/irods#6766]
### iRODS Version, OS and Version
iRODS 4.3.0, iCommands 4.3.0, centos7

### What did you try to do?
I tried to remove metadata attached by using wildcard option - `imeta rwm` 

```
[root@localhost irods]# imeta add -d test.py atr val
[root@localhost irods]# imeta ls -d test.py
AVUs defined for dataObj /hpc_icts/home/vsc33586/test.py:
attribute: atr
value: val
units:
```

irods info:
```
[root@localhost irods]# imiscsvrinfo
RCAT_ENABLED
relVersion=rods4.3.0
apiVersion=d
rodsZone=hpc_icts
up 1 days, 8:17
```
iCommands:
`iRODS Version 4.3.0                ihelp`

### Expected behavior
To see `rmw is deprecated` or `Unrecognized subcommand`

### Observed behavior (including steps to reproduce, if applicable)
```
[root@localhost irods]# imeta rmw -d test.py % %
[root@localhost irods]# imeta ls -d test.py
AVUs defined for dataObj /hpc_icts/home/vsc33586/test.py:
None
```

**Comment by korydraughn** [comment]
That is correct. No message is printed to the terminal, but the help text does mention deprecation.

See #6417

**Comment by mstfdkmn** [comment]
Ah okay thanks. Can we know possible removal time of `w(s)` in metadata?

**Comment by korydraughn** [comment]
We haven't discussed when the wildcard operations will be removed. You expect that to happen in a much later release.

The deprecation of the operation is there to signal that users should start to move away from using the wildcard operations.

**Comment by trel** [comment]
do we need an 'remove imeta rmw' issue?   I couldn't find one.   If not, this could be it?

**Comment by korydraughn** [comment]
I don't remember there being an issue for removal, so yes, I think we can use this issue for that.

**Comment by mstfdkmn** [comment]
Btw, a small remark: my main concern is how to implement the logical quota for our production environment, that is why asked this regarding an issue exists at the metadata guard plugin.

**Comment by trel** [comment]
renamed this issue.   not sure about `addw` or `lsw` either...

**Comment by korydraughn** [comment]
`addw` is also marked as deprecated.

**Comment by trel** [comment]
So, if we agree to removing all three - we should declare `lsw` deprecated beforehand as well.

**Comment by korydraughn** [comment]
I'm not sure we need to deprecate and remove `lsw` as it does not modify metadata. And given that it requires the client to specify a target, it can't bypass any permissions.

The only reason I can see for deprecating it is if someone wants to hide results based on some policy.

Are there other reasons for deprecating it?

**Comment by trel** [comment]
It would be the only wildcard remaining... and these subcommands are just 'shortcuts' for a proper genquery already.

So, mostly, consistency - removal of the wildcard handling at this level.

**Comment by korydraughn** [comment]
That's what I suspected.

I'm in favor of deprecating `lsw` too.

**Comment by trel** [comment]
We've deprecated all three (`addw` and `rmw` in 4.3.0 and `lsw` in 4.3.2).

I think removal can happen in 5.0.0.

**Comment by alanking** [comment]
`imeta adda` was deprecated in 4.3.0: https://github.com/irods/irods/issues/6187

I don't see any issue for removing it. Should that be done here as well?

**Comment by korydraughn** [comment]
Yes please.

**Comment by alanking** [comment]
`adda`, `lsw`, `addw`, and `rmw` have been removed. https://github.com/irods/irods/issues/8266 was created to completely remove the `rmw` internal implementation, but the external interface has been removed. Closing this.
